lxd/main_init.go

```diff
@@ -825,7 +825,7 @@ func (cmd *CmdInit) setProfileConfigItem(c lxd.ContainerServer, profileName stri
 // the auto/interactive modes.
 type cmdInitData struct {
 	api.ServerPut `yaml:",inline"`
-	Pools         []api.StoragePoolsPost
+	Pools         []api.StoragePoolsPost `yaml:"storage_pools"`
```

Comment:
Because if the name of structure field is called "Foo" the go yaml package will map it to "foo" when marshalling and unmarshalling. So "Networks" gets mapped to "networks" by default without an explicit annotation.

I can surely add annotations everywhere for extract clarity, if you prefer.